app/qml/banners/AutoHideBanner.qml

```diff
@@ -0,0 +1,31 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick 2.14
+import ".."
+
+Banner {
+  id: root
+
+  function show( interval = 3000 )
+  {
+    hideTimer.interval = interval
+    hideTimer.start()
+  }
+
+  showBanner: hideTimer.running
+
+  Timer {
+    id: hideTimer
+
+    interval: 3000
```

Comment:
3000 == cannot it be Banner property (so it is possible to set from outside). also constant used on 2 places